tests/test_cli.py

```diff
@@ -318,4 +356,4 @@ def test_correct_content_of_flag_file(self, mock_isfile, mock_link):
                 "05T00:43:05.826008Z, Instrument: SLSTR, Mode: EO, Satellite: Sentinel-3, Size: 415"
                 ".29 MB\n"
             )
-        )
+            )
```

Comment:
Since both parentheses are opened on the same line, please close them on the same line

Reply:
Done in the next commit